chart/values.schema.json

```diff
@@ -1351,6 +1351,14 @@
                         "type": "string"
                     }
                 },
+                "priorityClassName": {
+                    "description": "Set priority value from the priority class name",
```

Comment:
```suggestion
                    "description": "Specify priority for worker pods.",
```

This reads better to me?

Reply:
I just suggested it once, but this applies to all of them too.

Reply:
Updated all of them.